README.md

```diff
@@ -145,7 +169,7 @@ code, include a script or makefile that will do the job.
 
 ## A2.5
 
-«replace this with your answer»
+A2.5 folder contains the main.cpp file with implementation and test cases. Catch file contains the test functionalities. 
```

Comment:
Nice tests, and very nice code.

However, it looks as if the state machine in your code is pretty much taken from mine. I'm going to deduct 6 for that. If you can convince me that you came up with it independently, I'll add those marks back.

# -6